app/views/associate_ucr_summary.scala.html

```diff
@@ -55,7 +58,10 @@ <h2 id="master-consignment">@messages("associate.ucr.summary.masterConsignment")
                     <td class="associate_ucr__to" id="mucr-type">@messages("associate.ucr.summary.kind.mucr")</td>
                     <td id="mucr-reference">@mucr</td>
                     <td class="text-right associate_ucr__change">
-                        <a id="mucr-change" href="@consolidations.routes.MucrOptionsController.displayPage()">@messages("site.change")</a>
+                        <a id="mucr-change" href="@consolidations.routes.MucrOptionsController.displayPage()">
```

Comment:
Could you make id in a similar fashion as the previous one?

Reply:
Done.